packages/flame/test/game/game_widget/game_widget_test.dart

```diff
@@ -221,4 +238,242 @@ void main() {
       expect(game2, isNull);
     });
   });
+  group('focus', () {
+    testWidgets('autofocus starts focused', (tester) async {
+      final gameFocusNode = FocusNode();
+
+      await tester.pumpWidget(
+        GameWidget(
+          focusNode: gameFocusNode,
+          game: FlameGame(),
+          // ignore: avoid_redundant_argument_values
+          autofocus: true,
```

Comment:
```suggestion
```

Reply:
It's pretty good to keep this imho, it gives the reader of the test more context of what is being tested.

Reply:
Disagree, that is to explicitly show what that test tests.